spec/core_functions/color/grayscale.hrx

```diff
@@ -74,7 +74,49 @@ a {b: grayscale($color: white)}
 
 <===> named/output.css
 a {
-  b: white;
+  b:  white;
```

Comment:
@mirisuzanne 

For legacy colors, is it implied that the [conversion happens in `hsl`](https://github.com/sass/sass/blob/colors-2/proposal/color-4-new-spaces.md#colorgrayscale) but the output color is always returned in the origin space?

Reply:
With legacy colors, we don't guarantee that the output will be in a particular space. Generally the goal is that most existing functions will continue to work for legacy colors exactly the way they did before. So whatever grayscale currently returns with a legacy color is probably what it will keep returning.

Reply:
I think I might have been wrong about this, and we are planning to maintain the initial color space - which is rgb for hex and named colors. Might be worth confirming that with Natalie, though?